Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [workspace]
 resolver = "2"
 members = [
-    "engine-api",
-    "genesis-builder",
-    "moved",
-    "server",
+    "engine-api", "evm-ext", "genesis",
```

Comment:
Wow, this PR is a massive upgrade ✨

Why not each member go on to a new line?

Also curious, how come the unit test time on github action didn't improve? I'd think it would be improved too, but it still took 18 mins.

Reply:
Thanks for reviewing!

I added the crates via a command so I didn't notice.

From the latest job runs I can tell that it took 4-5 minutes off the job's time. However, the vast majority of it is not spent on running the tests but rather compiling + downloading dependencies, that's why the difference gets lost in that great amount of time.

Reply:
Ok just noticed the 22 min vs 18 min difference. We're making this change for local testing instead of Github improvement. So this work is appropriate 🙌